src/common/Tiles/MovieTile/index.js

```diff
@@ -14,11 +14,11 @@ import {
   Votes,
   MainInfo,
 } from "./styled";
-import star from "../../Images/star.svg";
+import star from "./star.svg";
```

Comment:
Why you don't want left all images in one folder and imported them in the needed komponents? This is the same SVG item.

Reply:
I changed it because I didn't see the point to left folder images just for one image